### DISCUSSION TITLE
Version 3.7.0 stops for cooling, even when cooling is disabled.

### DISCUSSION THREAD

**Discussion by e6910** [arkypita/LaserGRBL#1079] [Q&A]
Installed v 3.7.0. Jobs will move to starting position, but it will pause for cooling. Cooling is disabled.

Reverted back to 3.5.3, but the problem persists. You have to manually unpause to continue.

BTW it should be nice to name installer with a descriptive name and version number, like "laserGRBL-v030300.exe".

Just a suggestion...

**Comment by arkypita** [comment]
I have no evidence of this problem in the latest version, nor in previous versions.
Are you sure you have not enabled auto-cooling in the configuration?

A video of the issue can help
